src/pages/gift/about/AboutMain.js

```diff
@@ -0,0 +1,23 @@
+import { Container } from 'react-bootstrap';
```

Comment:
We can also consider adding a small link in the bottom of the landing pages for both claim and generate to this page. So users can follow the link to this page to learn more about gift process.

Reply:
True. Or maybe on the bottom of the card.